en/sql/schema/index_stmt.rst

```diff
@@ -227,6 +240,84 @@ Online unique index while other transactions inserts violates uniqueness
+    * The overflow page have a disadvantage in add, delete, and update performance, but are beneficial in search performance. Therefore, it is better to adjust the length of the overflow page for one key to be within tens or hundreds of pages, rather than setting the *deduplication level* high so that overflow pages are not created when creating an index with that option.
+
+
+There are two ways to specify *deduplicate level* when creating an index.
+
+Implicit method
+
+    This method automatically specifies *deduplicate level* when there is no explicit DEDUPLICATE option specified in the SQL statement. This method is affected by the set value of the system parameter **deduplicate_key_level**.
+    If **deduplicate_key_level** is greater than or equal to 1, *deduplicate level* is automatically set to **deduplicate_key_level**. 
+
+Explicit method
+
+    This is a method in which the user explicitly specifies the **DEDUPLICATE** option in the SQL statement. Regardless of the **deduplicate_key_level** set value, the *deduplicate level* specified by the user is applied.
+    As in the example below, specify the DEDUPLICATE statement directly.
+    
+    .. code-block:: sql
+    
+        CREATE TABLE tbl (a int default 0, b int, c int);
+        CREATE INDEX i_tbl_b on tbl (b) WITH DEDUPLICATE=3 COMMENT 'for deduplicate level 3';
+        CREATE INDEX i_tbl_b_c on tbl (b,c) WITH DEDUPLICATE=7 COMMENT 'for deduplicate level 1';
+
+.. warning::
+
+    * If **deduplicate_key_level** is **\-1**\, it is internally ignored and not applied even if it is specified explicitly. That is, in this case, all indexes are created with *deduplicate level* as **0**.
+    * A column name that begins with "**_dedup_**" cannot be created.
+
+.. note::
+
+    * When creating an index, if the composition of the key field is guaranteed to be UNIQUE, the DEDUPLICATE setting set by the user is ignored and *deduplicate level* is created as **0**.
+        * When the key field includes all key fields constituting a specific Primary Key or Unique Index
+        * However, except when used as an argument of a function
+
+Allowing multiple indexes on the same column in the same order
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+        
+    Multiple indexes can be defined with all key fields and filter conditions identical except for *deduplicate level*\.
+
+    .. code-block:: sql
+    
+        CREATE TABLE tbl (a int primary key, b int, c int);
+        CREATE INDEX idx1 ON tbl(b, c) WITH DEDUPLICATE=3;
+        CREATE INDEX idx2 ON tbl(b, c) WITH DEDUPLICATE=5;
+        CREATE UNIQUE INDEX idx_uk ON tbl(b); 
+        CREATE INDEX idx3 ON tbl(b, c) WITH DEDUPLICATE=7;
+
+    In the example above, idx1 and idx2 have the specified *deduplicate level*. However, since idx3 is guaranteed that column b is unique by idx_uk, *deduplicate level* is created as **0**, ignoring the user's designation.
```

Comment:
```suggestion
    In the example above, idx1 and idx2 have the specified *deduplicate level*. However, since idx3 is guaranteed by idx_uk that column b is unique, it is created *duplicate level* as **0** and ignore user's specified option.
```